### PR TITLE
chore(usage,metric): add pipeline release info in usage/metric data

### PR DESCRIPTION
**Pull request by heiruwu**
Because

- support pipeline release feature

This commit

- add pipeline release info in usage/metric data

### CODE REVIEW

**Comment by heiruwu** [comment]
Hi @donch1989 please take a look at this [commit](https://github.com/instill-ai/pipeline-backend/pull/234/commits/ead32a04c787379368c2c888fb2e3154f92860e3), I am not sure if this will pose any security concern by not using getUser* methods

File: pkg/service/service.go [hunk 1]
```diff
@@ -863,7 +863,17 @@ func (s *service) SetDefaultUserPipelineReleaseByID(ctx context.Context, ns reso
...
 
-func (s *service) triggerPipeline(ctx context.Context, ownerPermalink string, recipe *datamodel.Recipe, pipelineId string, pipelineUid uuid.UUID, pipelineInputs []*structpb.Struct, pipelineTriggerId string, returnTraces bool) ([]*structpb.Struct, *pipelinePB.TriggerMetadata, error) {
+func (s *service) triggerPipeline(
+	ctx context.Context,
+	ownerPermalink string,
+	recipe *datamodel.Recipe,
+	pipelineId string,
+	pipelineUid uuid.UUID,
+	pipelineReleaseId string,
+	pipelineReleaseUid string,
```
Comment (donch1989) [inline pkg/service/service.go:873]: I suggest that we use `uuid.UUID` type in the function params

Comment (heiruwu) [reply pkg/service/service.go:873]: will do

**Review by donch1989 — APPROVED** [review]
LGTM

**Pull request merged**